src/components/button/button.tsx

```diff
@@ -16,6 +15,7 @@ import {
   elButtonSizeMedium,
   elButtonSizeSmall,
 } from './__styles__'
```

Comment:
**todo:** I think we need to decide what the migration journey for consumers of the existing `ButtonGroup` will look like. What we want that journey to look like will influence whether we want to be exporting the new ButtonGroup from Button (like you are here) or if we want to continue exporting the now-deprecated ButtonGroup as-is, or if we should just delete the existing ButtonGroup entirely.

Reply:
Yes, I believe this will require a discussion with the team, or at least with you, to reach an agreement on the best approach.

Reply:
If it were up to me, I'd nuke the old one completely, but it's worth checking with Adrian (perhaps via the Elements dev chat on Teams) to see how he'd prefer the migration journey to look.